neuro_san/session/async_grpc_service_agent_session.py

```diff
@@ -97,7 +97,7 @@ async def function(self, request_dict: Dict[str, Any]) -> Dict[str, Any]:
             request_dict,
             service_messages.FunctionRequest())
 
-        self.umbrella_timeout.check_timeout()
+        Timeout.check_if_not_none(self.umbrella_timeout)
```

Comment:
Check the timeout if it's not None. Here and in 3 other places.
(Introduced a bug in previous checkin doh!)